compute/instances/how-to/migrate-volumes-snapshots-to-sbs.mdx

```diff
@@ -7,35 +7,67 @@ content:
   paragraph: This guide provides step-by-step instructions to help you migrate your existing Block Storage volumes and snapshots to Scaleway's new Block Storage management system.
 tags: instance snapshot volume b_ssd sbs
 dates:
-  validation: 2024-11-15
+  validation: 2025-01-14
   posted: 2024-08-07
 categories:
   - compute
 ---
 
 To enhance performance and reliability, Scaleway is transitioning the management of Block Storage volumes and snapshots from Compute to Storage.
 
-The migration process ensures that the characteristics of your volumes and snapshots remain unchanged, even for volumes attached to running Instances.
+To facilitate the transition to Scaleway's new Block Storage management, two main approaches for migrating your volumes exist, each with its unique benefits and drawbacks:
 
-After migration, these resources will be managed under [Storage](/storage/block/quickstart/) instead of Compute.
+- [Using the Instance API/CLI migration endpoint (Plan and Apply)](#migrating-using-the-migration-endpoint-of-the-instances-api-cli):
+   - **Advantage:** Migration can occur without service interruption, as the volume remains attached to the running Instance and fully accessible.
+   - **Drawback:** Only the volume's representation is migrated to the SBS API; the data itself doesn't move. This means the migrated volume will not benefit from low latency or higher IOPS, as it retains the original `b_ssd` characteristics.
 
-During migration, unified snapshots will be converted into [Block Storage Low Latency](/storage/block/) snapshots and removed from Compute. If a unified snapshot is part of an image, it will be replaced by the corresponding Block Storage Low Latency snapshot.
+- [Using the snapshot export/import features](#migrating-using-the-snapshot-exportimport-features):
```

Comment:
Shouldn't there be a slash in the anchor, as the title (## Migrating using the snapshot import/export features) does have it?

Reply:
Fixed anchor, no slash required after testing